src/App.js

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Home from './components/home';
+import Navbar from './components/navigation';
+import Quotes from './components/quotes';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/200794054-5a4536fc-6c7b-4b1f-9761-f709d115cae5.png)

- [ ] You have added a link to your license file but this link is broken :fearful: I guess you forgot to add the license file :smile: No worries you can add the license file using these [guildines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :blush: